src/routes/cart.router.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  addArticleToCart,
+  clearCart,
+  createCart,
+  getCart,
+  quantityArticleCart,
+  removeArticleFromCart,
+} from '@controlers/cart.controller';
+import { isAuth } from '@utils/isAuth';
+import { Router } from 'express';
+
+const cartRouter = Router();
+
+cartRouter.get('/cart', isAuth, getCart);
+cartRouter.post('/cart', isAuth, createCart);
+cartRouter.post('/add-cart', isAuth, addArticleToCart);
+cartRouter.post('/quantity-cart', isAuth, quantityArticleCart);
+cartRouter.delete('/cart', isAuth, clearCart);
```

Comment:
same as create cart. we dont need this option, cart should be removed once all products from cart are removed